index.bs

```diff
@@ -5144,7 +5144,7 @@ The [=type name=] of the
 
 The {{unrestricted float}} type is a floating point numeric
 type that corresponds to the set of all possible single-precision 32 bit
-IEEE 754 floating point numbers, finite and non-finite.  [[!IEEE-754]]
+IEEE 754 floating point numbers, finite, non-finite, and special "not a number" values (NaNs). [[!IEEE-754]]
```

Comment:
This has been a point of contention in TC39 for a while, as browsers don't follow the part of the spec that says they should be indistinguishable, so we are trying to update the spec. For example, the following code returns two distinct NaNs in Firefox: https://jsbin.com/noface/edit?html,console